addon/helpers/list.js

```diff
@@ -1,7 +1,24 @@
 import { MicroState } from 'ember-microstates';
+import isInteger from '../utils/is-integer';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
```

Comment:
Can we move this up to parameter declaration? 

`initialValueFor([array = []])`